projects/pt1/python/torch_mlir_e2e_test/onnx_backends/linalg_on_tensors.py

```diff
@@ -0,0 +1,66 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+
+from torch_mlir.compiler_utils import run_pipeline_with_repro_report
+from torch_mlir.ir import *
+from torch_mlir.passmanager import *
+from torch_mlir.torchscript import OutputType
+from torch_mlir.torchscript import _lower_mlir_module
+
+from torch_mlir_e2e_test.linalg_on_tensors_backends.refbackend import RefBackendLinalgOnTensorsBackend
+
+from .abc import OnnxBackend
+
+__all__ = [
+    "LinalgOnTensorsOnnxBackend",
+]
+
+# The pipeline of func.func passes that lower the ONNX backend contract to the
+# Linalg-on-Tensors backend contract accepted by RefBackend.
+ONNX_TO_TORCH_FUNC_PIPELINE = ",".join([
+    "convert-torch-onnx-to-torch",
+])
+
+
+class LinalgOnTensorsOnnxBackend(OnnxBackend):
+    """Main entry-point for the linalg-on-tensors based ONNX backend.
+
+    This currently uses the linalg-on-tensors RefBackend for actual execution.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.refbackend = RefBackendLinalgOnTensorsBackend()
+
+    def compile(self, imported_module: Module):
+        """Compiles an imported module that satisfied the ONNX backend contract.
+
+        Args:
+          imported_module: The MLIR module consisting of funcs in the ONNX
+            dialect.
```

Comment:
Is "the ONNX dialect" accurate, or is this really the "torch" dialect with ONNX ops? (I'm not sure what terminology is used for that in torch-mlir)

Reply:
Updated description.